scripts/lc_marc_update.py

```diff
@@ -102,7 +102,9 @@ def read_block(block):
     ftp.close()
     sys.exit(0)
 
-bad = open(c['log_location'] + 'lc_marc_bad_import', 'a')
+with open(c['log_location'] + 'lc_marc_bad_import', 'a') as bad:
```

Comment:
We might want to reevaluate where this line should be. I don't think it makes too much sense to have it like this. Perhaps closer to where we use bad.

Reply:
fixed in the last commit!